Sources/NIOPosix/SelectorKqueue.swift

```diff
@@ -82,9 +82,17 @@ extension KQueueEventFilterSet {
             return UInt16(self.contains(event) ? EV_ADD : EV_DELETE)
         }
 
-        for (event, filter) in [
+        #if os(FreeBSD)
+        let eventFilterPairs = [
+            (KQueueEventFilterSet.read, EVFILT_READ), (.write, EVFILT_WRITE)
```

Comment:
Why was `EVFILT_EXCEPT` removed? The FreeBSD manual pages suggest it's present.

Reply:
Interesting, this is likely to cause knock-on test failures due to our inability to replicate our existing behaviour on `readEOF`.